src/ORM/Table.php

```diff
@@ -321,17 +321,17 @@ public function initialize(array $config)
     }
 
     /**
-     * Returns the database table name or sets a new one
+     * Gets/Sets the database table name.
      *
      * @param string|null $table the new table name
      * @return string
      */
     public function table($table = null)
     {
-        if ($table !== null) {
+        if (func_num_args() === 1) {
             $this->_table = $table;
         }
-        if ($this->_table === null) {
+        if (func_num_args() === 0) {
```

Comment:
Entirely true, there are probably more errors. I will improve whatever you add to concerns tomorrow.